app/resources/views/includes/header.blade.php

```diff
@@ -1,5 +1,5 @@
 		<div class="navbar-inner">
-			<a id="logo" href="/">AccessLocator</a>
+			<a id="logo" href="/">AccessLocatorrr</a>
```

Comment:
Correct this.  Make the spelling AccessLocator instead of repeating the "r" at the end.